qa/integration-tests/src/test/java/io/camunda/zeebe/it/gateway/GatewayAuthenticationIdentityIT.java

```diff
@@ -189,15 +177,13 @@ void deployModelSucceedsWithValidAuthToken() {
                     .build())
             .build()) {
       // when
-      final var deploymentFuture =
-          client
-              .newDeployResourceCommand()
-              .addProcessModel(PROCESS_MODEL, "model.bpmn")
-              .send()
-              .toCompletableFuture();
+      final var deploymentFuture = client.newTopologyRequest().send().toCompletableFuture();
 
       // then
-      assertThat(deploymentFuture).succeedsWithin(Duration.ofSeconds(1));
+      assertThat(deploymentFuture).succeedsWithin(FIRST_REQUEST_TIMEOUT);
+      // second request should be faster, given the first request warmed up the token validation
+      assertThat(client.newTopologyRequest().send().toCompletableFuture())
+          .succeedsWithin(Duration.ofMillis(100));
```

Comment:
:wrench: CI can be sometimes too slow. So it might be better to keep the timeout at 1s to reduce chance of flakyness.

Reply:
better safe than flaky 🙃